frame/staking/src/benchmarking.rs

```diff
@@ -593,7 +618,7 @@ benchmarks! {
 			T::MaxNominatorRewardedPerValidator::get() as u32,
 			false,
 			true,
-			RewardDestination::Staked,
+			PayoutDestination::Stake,
```

Comment:
We should benchmark this with worst case scenario: each nominator for a validator has setup split rewards which will imply double the number of payouts compared to today. I remember some concerns that reward payouts already leads to overweight blocks, so may be good to crunch some numbers around it.